masterlist.yaml

```diff
@@ -3077,6 +3332,61 @@ plugins:
       - Delev
       - Invent
       - Relev
+  - name: 'Lore Weapon Expansion.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/9660/']
+    tag:
+      - Delev
+      - Relev
+    clean:
+      - crc: 0x0D7685E5 # v1.4a
+        util: SSEEdit v3.2.2
+  - name: 'Lore Weapon Expansion - Daedric Crescent.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/9660/']
+    group: *earlyLoadersGroup
```

Comment:
Main file is not a master of optional files.
They're loaded early to avoid reverting Cell changes.